io/disk/htx_block_devices.py

```diff
@@ -59,7 +63,12 @@ def setUp(self):
             for disk in self.block_devices.split():
                 self.block_device.append(disk.rsplit("/")[-1])
             self.block_device = " ".join(self.block_device)
+        self.failed = 0
```

Comment:
Minor comment: better to use bool?

Reply:
Sure, I first thought of having many failure values and dealing each one separately, but that was not needed. Will change it into boolean